kuma/users/tests/test_middleware.py

```diff
@@ -24,3 +24,6 @@ def test_ban_middleware(self):
 
         resp = self.client.get('/')
         self.assertTemplateUsed(resp, 'users/user_banned.html')
+        expires_datetime = datetime.strptime(resp['Expires'],
+                                             '%a, %d %b %Y %H:%M:%S %Z')
+        assert expires_datetime.timetuple() <= gmtime()
```

Comment:
`email.utils.parsedate` can parse these RFC 2822 dates giving something you can pass to `time.mktime` and compare it from there? Might save this ugly strptime format string?